Assets/_BuildingBlocks/Credentials/Scripts/StoredAuthorization/UsernamePassword.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.Credentials.StoredAuthorization
+{
+    [Serializable]
+    public class UsernamePassword : HeaderBasedAuthorization
+    {
+        public string username;
```

Comment:
Hoofdletter in Username, en zou deze - net als Password niet een get-only property moeten zijn?